core/controllers/suggestion.py

```diff
@@ -246,8 +248,8 @@ def get(self, target_type, suggestion_type):
             suggestions = suggestion_services.get_reviewable_suggestions(
                 self.user_id, suggestion_type)
             self._render_suggestions(target_type, suggestions)
-        except Exception as e:
-            raise self.InvalidInputException(e)
+        except Exception:
+            raise self.InvalidInputException(traceback.format_exc())
```

Comment:
@sagangwee, will it be fine to remove the try-catch from here?

Reply:
Yeah I think it's fine to remove the try-catch here actually, since we're not performing any extra handling (we're just re-raising any thrown exceptions).

Reply:
Thanks @DubeySandeep for the feedback and thanks @sagangwee for your thoughts :) I'll remove the try-catch.